src/App/Containers/Users/UsersContainer.js

```diff
@@ -1,57 +1,77 @@
 import React, { Component } from 'react'
-
 import SearchModule from '../../components/Search/index.search'
 import UsersTableModule from '../Users/TableUsers'
-import ModalModule from './ModalUsers'
-import { Button } from 'reactstrap'
+// import ModalModule from './ModalUsers'
+// import { Button } from 'reactstrap'
+
+import Pagination from "react-js-pagination"
+// require("bootstrap/less/bootstrap.less");
```

Comment:
As per the discussion, try to avoid the pagination package and try to implement our own. It will help in the long run.